src/order/router.js

```diff
@@ -0,0 +1,36 @@
+const { Router } = require("express");
+const GetOrdersRequestSchema = require("./schema");
+const { GetOrdersRequest } = require("./models");
+
+const router = Router();
+
+module.exports = router;
+
+const signer = require("../helpers/signer.service").init(process.env.KEY);
+const orderService = require("./service").init(process.env.REPORTING_API_URL);
+
+const secretsService = {
+  reportingApiUrl: () => process.env.REPORTING_API_URL,
+  key: () => process.env.KEY,
+  publisherToken: () => process.env.PUBLISHER_TOKEN,
+};
+
+router.post("/", async (req, res) => {
+  const { error } = GetOrdersRequestSchema.validate(req.body);
+  if (error) {
+    return res.status(422).json({ error: error.details[0].message });
+  }
+
+  const getOrdersRequest = GetOrdersRequest.fromJson(req.body);
+  const signature = signer.createSignature(
```

Comment:
I believe we removed the signature when sending requests to reporting api and assets upload services

Reply:
I can remove it from this repo in these two places, do you know other places where it should be removed?